win32ss/user/user32/controls/button.c

```diff
@@ -1196,7 +1196,11 @@ static void BUTTON_CheckAutoRadioButton( HWND hwnd )
     do
     {
         if (!sibling) break;
+#ifdef __REACTOS__
+        if ((SendMessageW( sibling, WM_GETDLGCODE, 0, 0 ) & (DLGC_BUTTON | DLGC_RADIOBUTTON)) == (DLGC_BUTTON | DLGC_RADIOBUTTON))
+#else
         if (SendMessageW( sibling, WM_GETDLGCODE, 0, 0 ) == (DLGC_BUTTON | DLGC_RADIOBUTTON))
```

Comment:
Is
https://github.com/reactos/reactos/blob/f0179741d1a5aa67ee62a0f33b07962b63a5fad1/win32ss/user/user32/windows/dialog.c#L2631
affected too?

Reply:
Thanks.